src/xo-app/home/pool-item.js

```diff
@@ -0,0 +1,144 @@
+import _ from 'intl'
+import Component from 'base-component'
+import Ellipsis, { EllipsisContainer } from 'ellipsis'
+import flatMap from 'lodash/flatMap'
+import Icon from 'icon'
+import map from 'lodash/map'
+import React from 'react'
+import reduce from 'lodash/reduce'
+import SingleLineRow from 'single-line-row'
+import size from 'lodash/size'
+import Tags from 'tags'
+import Tooltip from 'tooltip'
+import { BlockLink } from 'link'
+import { Row, Col } from 'grid'
+import { Text } from 'editable'
+import {
+  addTag,
+  editPool,
+  getHostMissingPatches,
+  removeTag
+} from 'xo'
+import {
+  connectStore,
+  formatSize
+} from 'utils'
+import {
+  createGetObjectsOfType,
+  createSelector
+} from 'selectors'
+
+import styles from './index.css'
+
+@connectStore(() => {
+  const getPoolHosts = createGetObjectsOfType('host').filter(createSelector(
+    (_, props) => props.item.id,
+    poolId => host => host.$pool === poolId
+  ))
+
+  const getMissingPatches = createSelector(
+    getPoolHosts,
+    hosts => {
+      return Promise.all(map(hosts, host => getHostMissingPatches(host)))
+      .then(patches => flatMap(patches))
+    }
+  )
+
+  return {
+    hosts: getPoolHosts,
+    missingPaths: getMissingPatches
+  }
+})
+export default class PoolItem extends Component {
+  _addTag = tag => addTag(this.props.item.id, tag)
+  _removeTag = tag => removeTag(this.props.item.id, tag)
+  _setNameDescription = nameDescription => editPool(this.props.item, { name_description: nameDescription })
+  _setNameLabel = nameLabel => editPool(this.props.item, { name_label: nameLabel })
+  _toggleExpanded = () => this.setState({ expanded: !this.state.expanded })
```

Comment:
It's indeed probably not easy and the current need is trivial enough to keep it as is.